Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -16,7 +16,7 @@ final class WebSocket {
     /// A WebSocket connection callback.
     private let onEvent: (Event) -> Void
     private var onEventObservers = [String: Client.OnEvent]()
-    private var provider: WebSocketProvider
+    var provider: WebSocketProvider
```

Comment:
I don't think we should do that

Reply:
For the latest changes, I need to make internal to make possible to change WebSocket request.